.circleci/config.yml

```diff
@@ -9,11 +9,13 @@ default: &default-steps
     - run: make elpa
     - run: emacs --version
     - run: cask --version
+    - run: make unit
```

Comment:
I think a target like `unit-tests` would be better here. `make test` should probably run all tests and there should be a separate `make` target for the integration tests.